index.html

```diff
@@ -0,0 +1,637 @@
+        </nav>
+
+      </div>
+
+    </header>
+
+    <section>
+      <h1>hi</h1>
+      <div class="container-fluid main">
+        <div class="row d-flex justify-content-between">
+          <div class="col-lg-4  col-md-6  col-sm-12 d-flex featured flex-column">
+            <div class="featured1">
+              <h4 class="heading">FEATURED STORIES</h4>
+              <img src="images/bernie.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p class="featured-p">"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+            </div>
+            <div class="featured1">
+              <img src="images/3.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+            </div>
+
+            <div class="featured1">
+              <img src="images/4.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+            </div>
+
+            <div class="featured1">
+
+              <img src="images/5.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+
+            <div class="featured1">
+
+              <img src="images/7.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+
+            <div class="featured1">
+              <img src="images/9.png" alt="">
+              <p><a href="#">Bernie Sanders on Trump's Coronavirus Response: 'Shut This President Up'</a></p>
+              <p>"He is undermining the doctors and scientists who are trying to help the American people," the Vermont senator said during Sunday's Democratic debate.</p>
+
+            </div>
+          </div>
+          <div class="col-lg-4 col-md-6 top d-flex flex-column">
+            <h4 class="heading">TOP STORY</h4>
+            <img src="images/federal.png" alt="">
+            <p><a href="#">Federal Reserve Slashes Interest Rates to Near 0% Amid Coronavirus Pandemic</a></p>
```

Comment:
- [ ] This type of  title should behave like the one I commented on the Featured Section.